### PR TITLE
Update assembly file version to 6.0

### PR DESCRIPTION
**Pull request by BHoMBot**
Update assembly file version to 6.0

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check project-compliance

File: Acoustic_oM/Acoustic_oM.csproj [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
     <Company />
     <Authors>BHoM</Authors>
-        <Description>https://github.com/BHoM/BHoM</Description>
+            <Description>https://github.com/BHoM/BHoM</Description>
     <Copyright>Copyright © https://github.com/BHoM</Copyright>
-    <AssemblyVersion>5.0.0.0</AssemblyVersion>
-    <FileVersion>5.3.0.0</FileVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <FileVersion>6.0.0.0</FileVersion>
     <Version>5.0.0</Version>
```
Comment (IsakNaslundBh) [inline Acoustic_oM/Acoustic_oM.csproj:11]: Question on this. What does this version reference?

**Comment by FraserGreenroyd** [comment]
@BHoMBot check project-compliance

**Review by FraserGreenroyd — APPROVED** [review]
Approving from a DevOps perspective after code review.

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance
@BHoMBot check versioning
@BHoMBot check core
@BHoMBot check null-handling
@BHoMBot check serialisation
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: branch-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**